utils/unittest/fixtures.go

```diff
@@ -46,3 +48,18 @@ func StorableTransactionResultFixture() types.StorableTransactionResult {
 		},
 	}
 }
+
+func SDKEventToFlow(event sdk.Event) (flowgo.Event, error) {
```

Comment:
Why was this function moved out of the `convert` package? The inverse function is still there.

The function is exported and there might be downstream dependencies that use it.

Reply:
This function is only used for testing in this package.  I can move it back if it is needed outside of tests in this repo.

Reply:
That would be great, thanks!